test.json

```diff
@@ -0,0 +1,1178 @@
+    "dont_build": null,
+    "lifecycle": "finished",
+    "fleet": "picard",
+    "stop_time": "2018-02-06T04:23:37.763Z",
+    "build_time_millis": 48560,
+    "start_time": "2018-02-06T04:22:49.203Z",
+    "platform": "2.0",
+    "outcome": "success",
+    "vcs_url": "https://github.com/jesims/jesi",
+    "author_name": "André de Jager",
+    "queued_at": "2018-02-06T04:22:43.897Z",
+    "author_email": "andrethehunter@gmail.com"
+  },
+  {
+    "committer_date": "2018-02-06T11:35:37+10:00",
+    "body": "",
+    "usage_queued_at": "2018-02-06T01:47:22.235Z",
+    "reponame": "jesi",
+    "build_url": "https://circleci.com/gh/jesims/jesi/8589",
+    "parallel": 1,
+    "branch": "JESI-2007",
+    "username": "jesims",
+    "author_date": "2018-02-06T11:35:37+10:00",
+    "why": "github",
+    "user": {
+      "is_user": true,
+      "login": "axrs",
+      "avatar_url": "https://avatars2.githubusercontent.com/u/580014?v=4",
+      "name": "Alexander R Scott",
+      "vcs_type": "github",
+      "id": 580014
+    },
+    "vcs_revision": "719332fd0cf610461a2fa279d9d949961e7409b5",
+    "workflows": {
+      "job_name": "merge_into_develop",
+      "job_id": "064ba350-b498-45c6-bbd9-397617ff84e9",
+      "workflow_id": "5b45a49d-ca99-44c0-8f87-f12d5b2146bb",
+      "workspace_id": "5b45a49d-ca99-44c0-8f87-f12d5b2146bb",
+      "upstream_job_ids": [
+        "c2fe3ccb-c32d-44e6-8b0f-251b9a4a7bd6",
+        "98bf3a3b-e23f-405e-85c5-fe3171dc1805",
+        "6176ca54-2a01-460e-b335-a721420bd024",
+        "3463c45e-e038-4719-aa1f-58b8fdea56bc"
+      ],
+      "upstream_concurrency_map": {
+        "3463c45e_e038_4719_aa1f_58b8fdea56bc": [
+          "6176ca54-2a01-460e-b335-a721420bd024",
+          "98bf3a3b-e23f-405e-85c5-fe3171dc1805"
+        ],
+        "6176ca54_2a01_460e_b335_a721420bd024": [
+          "3463c45e-e038-4719-aa1f-58b8fdea56bc",
+          "98bf3a3b-e23f-405e-85c5-fe3171dc1805"
+        ],
+        "98bf3a3b_e23f_405e_85c5_fe3171dc1805": [
+          "3463c45e-e038-4719-aa1f-58b8fdea56bc",
+          "6176ca54-2a01-460e-b335-a721420bd024"
+        ]
+      },
+      "workflow_name": "build_test_merge"
+    },
+    "vcs_tag": null,
+    "pull_requests": [
+      {
+        "head_sha": "9b0efeace3d10eb3bc1b38b60233d53011f0ecef",
+        "url": "https://github.com/jesims/jesi/pull/697"
+      }
+    ],
+    "build_num": 8589,
+    "committer_email": "xander@cadcoder.com",
+    "status": "success",
+    "committer_name": "Alexander Scott",
+    "subject": "Add db require",
+    "dont_build": null,
+    "lifecycle": "finished",
+    "fleet": "picard",
+    "stop_time": "2018-02-06T01:47:33.605Z",
+    "build_time_millis": 9609,
+    "start_time": "2018-02-06T01:47:23.996Z",
+    "platform": "2.0",
+    "outcome": "success",
+    "vcs_url": "https://github.com/jesims/jesi",
+    "author_name": "Alexander Scott",
+    "queued_at": "2018-02-06T01:47:22.255Z",
+    "author_email": "xander@cadcoder.com"
+  }
+]
```

Comment:
There's no private information in here right?

Reply:
Nothing really scary. A private github link to pull requests and commit hashes. Some random commit messages with JIRA numbers. etc.

I'll clean and sanitise